.github/markdownlint.yml

```diff
@@ -0,0 +1,9 @@
+# Markdownlint configuration file
+default: true,
+line-length: false
```

Comment:
This is not up2date anymore ;-) 

```
# Markdownlint configuration file
default: true,
line-length: false
no-duplicate-header:
    siblings_only: true

```

Reply:
the file seems up to date with the template in `nfcore/tools`.